sympy/core/power.py

```diff
@@ -352,6 +352,15 @@ def _eval_is_zero(self):
                     return self.exp.is_positive
                 elif (1 - abs(self.base)).is_negative:
                     return self.exp.is_negative
+        else:
+            # when self.base.is_zero is None
+            if self.exp.is_finite:
+                return None
+            elif self.exp.is_infinite:
+                if (1 - abs(self.base)).is_positive:
+                    return self.exp.is_positive
+                elif (1 - abs(self.base)).is_negative:
```

Comment:
I didn't put much thought on this part. Just put it here to not break anything.
It looks like it won't be necessary. Thanks!